### PR TITLE
1. Created a config.properties file and added some values in it to use in

### PR DESCRIPTION
**Pull request by BahubaliPR**
program to remove hard coded values.

2. Created some packages and created dummy class files for time being.
these packages will be using in my project in future.

Please review the code and provide the comments.

### CODE REVIEW

File: src/main/java/org/utest/com/base/Base.java [hunk 1]
```diff
@@ -4,21 +4,33 @@
...
 import org.utest.com.extentreports.ExtentReport;
 import org.utest.com.utility.ExcelReader;
+import org.utest.com.webdriverlisteners.WebEventListener;
 
 public class Base {
 
+	/*
+	 * Creating reference for EventFirinfWebDriver and WebEventLestener
+	 */
+	public EventFiringWebDriver eventDriver = null;
```
Comment (gskr442) [inline src/main/java/org/utest/com/base/Base.java]: What does EventFireWebDriver do?

File: src/main/java/org/utest/com/alerts/AlertsHandler.java [hunk 2]
```diff
@@ -0,0 +1,8 @@
+package org.utest.com.alerts;
+
+/**
+ * Bahubali P R
+ */
+public class AlertsHandler {
```
Comment (gskr442) [inline src/main/java/org/utest/com/alerts/AlertsHandler.java]: Separate class For Alert Handler?

File: src/main/java/org/utest/com/assertions/AssertionHandler.java [hunk 3]
```diff
@@ -0,0 +1,8 @@
+package org.utest.com.assertions;
+
+/**
+ * Bahubali P R
+ */
+public class AssertionHandler {
```
Comment (gskr442) [inline src/main/java/org/utest/com/assertions/AssertionHandler.java]: Separate class for AssertionHandler?

File: src/main/java/org/utest/com/wait/WebDriverWaits.java [hunk 4]
```diff
@@ -1,8 +1,68 @@
...
+	 */
+	public void waitUntilPageLoad(WebDriver driver) {
+		driver.manage().timeouts().pageLoadTimeout(implicitWaitTime, TimeUnit.SECONDS);
+	}
+
+	/*
+	 * Waits until the element to be clickable on the current page
+	 */
+	public void waitUntilElementToBeClickable(WebElement element, WebDriver driver) {
+		wait = new WebDriverWait(driver, 30);
```
Comment (gskr442) [inline src/main/java/org/utest/com/wait/WebDriverWaits.java]: Is it different than other wait conditions ? If not so, remove the number and give a variable "explicitWaitTime".

File: src/main/java/org/utest/com/windows/WindowHandler.java [hunk 5]
```diff
@@ -1,8 +1,32 @@
 package org.utest.com.windows;
 
+import java.util.Set;
+
```
Comment (gskr442) [inline src/main/java/org/utest/com/windows/WindowHandler.java]: Remove Space.

File: src/main/java/org/utest/com/wait/WebDriverWaits.java [hunk 6]
```diff
@@ -1,8 +1,68 @@
...
+import org.utest.com.configfilereader.PropertiesFileReader;
+
 /**
  * Bahubali P R
  */
 public class WebDriverWaits {
 
+	private WebDriverWait wait = null;
+	private FluentWait<WebDriver> fwait = null;
+
```
Comment (gskr442) [inline src/main/java/org/utest/com/wait/WebDriverWaits.java]: Remove Space.

File: src/main/java/org/utest/com/base/Base.java (hunk 1 above)
Comment (BahubaliPR) [reply src/main/java/org/utest/com/base/Base.java]: Removed EventFiringWebDriver reference and also removed extra spaces.

File: src/main/java/org/utest/com/alerts/AlertsHandler.java (hunk 2 above)
Comment (BahubaliPR) [reply src/main/java/org/utest/com/alerts/AlertsHandler.java]: Removed this alert handler class.

File: pom.xml [hunk 7]
```diff
@@ -8,6 +8,32 @@
 	<name>utest</name>
 	<url>http://maven.apache.org</url>
 
+	<!-- <build>
```
[the remark this answers is not in the document]
Comment (BahubaliPR) [reply pom.xml]: Yes this is required in the future.

File: src/main/java/org/utest/com/wait/WebDriverWaits.java (hunk 4 above)
Comment (BahubaliPR) [reply src/main/java/org/utest/com/wait/WebDriverWaits.java]: Removed hard coded value and getting value from the properties file.

File: src/main/java/org/utest/com/windows/WindowHandler.java (hunk 5 above)
Comment (BahubaliPR) [reply src/main/java/org/utest/com/windows/WindowHandler.java]: Removed spaces from WindowHandler.java class.

File: src/main/java/org/utest/com/wait/WebDriverWaits.java (hunk 6 above)
Comment (BahubaliPR) [reply src/main/java/org/utest/com/wait/WebDriverWaits.java]: Removed spaces from the WebDriverWaits class.

File: pom.xml (hunk 7 above)
[the remark this answers is not in the document]
Comment (gskr442) [reply pom.xml]: Okay.

**Pull request merged**